.snippets/code/smart-contracts/libraries/web3-js/updateStorage.js

```diff
@@ -64,13 +54,12 @@ const updateStorage = async (config) => {
   }
 };
 
-// Example usage
 const config = {
```

Comment:
This config isn't being used anywhere - why do we need it? Should it be same as config in the `deploy.js` file where you've recently added `name` and `chainId`?